docs/components/modeler/dmn/dmn.md

```diff
@@ -0,0 +1,62 @@
+---
+id: camunda-modeler-dmn
+title: Editing DMN in Camunda Modeler
```

Comment:
Do you think we should add (Coming soon!) or something in the title? I'm thinking of someone looking at the sidebar and trying to help them understand as we add content in there, what that experience may look like?

Reply:
Please add a Platform-only label to this instead of Coming Soon.